x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -1107,28 +1106,14 @@
     "kbn.dashboard.listing.dashboardsTitle": "仪表板",
     "kbn.dashboard.listing.table.descriptionColumnName": "描述",
     "kbn.dashboard.listing.table.titleColumnName": "标题",
-    "kbn.dashboard.panel.customizePanel.displayName": "定制面板",
```

Comment:
maybe you want to just rename the ids to the new ones here, its not mandatory however.

Reply:
This is all done automatically with a script.  `node scripts/i18n_check.js --fix` i think is what I ran.